Havit.Blazor.Components.Web.Bootstrap/Navigation/SidebarCollapsedContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Havit.Blazor.Components.Web.Bootstrap;
+
+/// <summary>
+/// Context indicating whether a <see cref="HxSidebar"/> is collapsed or expanded.
+/// </summary>
+public class SidebarCollapsedContext
+{
+	/// <summary>
+	/// Indicates whether a <see cref="HxSidebar"/> is collapsed or expanded.
+	/// </summary>
+	public bool Collapsed { get; set; }
```

Comment:
We have to distinguish horizontal and vertical motion in the sidebar.
My proposal for property names:
* `SidebarCollapsed`
* `ItemExpanded`

The Sidebar is expanded by default, therefor `SidebarCollapsed` makes sense (`if (context.SidebarCollapsed) ...`).
The SidebarItem is collapsed by default therefor `ItemExpanded` makes sense...

What do you think?
cc @crdo @alexax578

Reply:
Seems reasonable, will adjust the naming.